dlt/extract/extractors.py

```diff
@@ -215,13 +217,26 @@ class ObjectExtractor(Extractor):
 class ArrowExtractor(Extractor):
     """Extracts arrow data items into parquet. Normalizes arrow items column names.
     Compares the arrow schema to actual dlt table schema to reorder the columns and to
-    insert missing columns (without data).
+    insert missing columns (without data). Adds _dlt_load_id column to the table if
+    `add_dlt_load_id` is set to True in normalizer config.
 
     We do things that normalizer should do here so we do not need to load and save parquet
     files again later.
 
+    Handles the following types:
+    - `pyarrow.Table`
+    - `pyarrow.RecordBatch`
+    - `pandas.DataFrame` (is converted to arrow `Table` before processing)
     """
 
+    # Inject the parts of normalize configuration that are used here
+    @with_config(
+        spec=ItemsNormalizerConfiguration, sections=(known_sections.NORMALIZE, "parquet_normalizer")
+    )
+    def __init__(self, *args: Any, add_dlt_load_id: bool = False, **kwargs: Any) -> None:
```

Comment:
This was my solution to support the old normalize config.  
I think it makes sense to keep it rather than move to `extract` section and have consistent with object normalizer.

Reply:
hmmm idea is very good.. but maybe you could decorate a method in this class. not __init__? so you call it just to retrieve config,